skimage/color/colorconv.py

```diff
@@ -1681,16 +1683,21 @@ def lab2lch(lab, *, channel_axis=-1):
     Returns
     -------
     out : (..., 3, ...) ndarray
-        The image in LCH format, in a N-D array with same shape as input `lab`.
+        The image in CIE-LCh color space, of same shape as input.
 
     Raises
     ------
     ValueError
-        If `lch` does not have at least 3 color channels (i.e. l, a, b).
+        If `lab` does not have at least 3 channels (i.e., L*, a*, and b*).
 
     Notes
     -----
-    The Hue is expressed as an angle between ``(0, 2*pi)``
+    The h channel (i.e., hue) is expressed as an angle in range ``(0, 2*pi)``.
```

Comment:
Do the `L` and `C` also have a value range?

Reply:
They do, but here's my understanding:

* The priority/necessity is to document the *input* ranges (hence my wording in https://github.com/scikit-image/scikit-image/issues/1185#issuecomment-1399319643 -- to be edited/improved?) because the user needs to know for sure whether the data they are passing makes sense to the function;
* Of course, it wouldn't hurt to document the *output* ranges as well, but this lacking is not as much of a showstopper: Presumably, when calling `lab2lch`, the user has a clear intention and means "I want to land in cylindrical CIE-LAB" and, should any doubts arise, maybe most can be cleared upon inspecting the output;
* So, in this example, the user would expect h to be an angle, and the note is only here to specify that the angle is expressed in radians `(0, 2*pi)` as opposed to degrees `(0, 360)`. In this sense, it is not analogous to L or C.

Reply:
@lagru here's my suggestion: Let's document the input ranges and link to the inverse function (under "See also?") so no information is missing, but it's not too redundant either. What do you think?

Reply:
Sound's perfect!

Reply:
Oh, and by the way, I really like how you engage with feedback and in discussions! You are really clear and transparent about your thoughts and line of reasoning which is all kinds of helpful. Just wanted to point that out. :blush:

Reply:
Thank you, @lagru! I'm glad that my sharing my thought process is valuable to you. :slightly_smiling_face: 

I believe I have addressed all of your review comments, so this should now be ready for a final review. Thanks again for your thorough review!